coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,35 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.InvalidFilterException import InvalidFilterException
+
+from coalib.parsing.filters.LanguageFilter import language_filter
+from coalib.parsing.filters.CanDetectFilter import can_detect_filter
+from coalib.parsing.filters.CanFixFilter import can_fix_filter
+
+
+class FilterHelper:
```

Comment:
hm kinda interesting: You never use an instance of this class :D (makes also no sense as there are no class members :D)

Up to you: Maybe moving all functions to module scope^^